### PR TITLE
Volume Projection Tutorial

### PR DESCRIPTION
**Pull request by garrettwrong**
Researcher requested an example of using ASPIRE tools to generate volume projects using prescribed rotations.

This is a draft for internal review/discussion.  There are similarities between this and some blocks of other tutorials.

### CODE REVIEW

File: gallery/tutorials/generating_volume_projections.py [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+    dtype=v.dtype,  # match our datatype to the Volume.
+    noise_filter=white_noise_filter,  # optionally prescribe noise
+)
+
+# %%
+# Yield projection images from the Simulation Source
+# --------------------------------------------------
+
+# Consume images from the source by providing
+# a starting indexand number of images.
```
Comment (j-c-c) [inline gallery/tutorials/generating_volume_projections.py]: Missing a space "index and"

File: gallery/tutorials/generating_volume_projections.py [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+# Here we generate the first 3 and peek at them.
+src.images(0, 3).show()
+src.projections(0, 3).show()
```
Comment (j-c-c) [inline gallery/tutorials/generating_volume_projections.py:114]: Not sure if this is of much concern, but these images appear a bit small and create a small thumbnail image. If it's something we care about it looks like we can change `figsize=(width,height)`

**Review by j-c-c — CHANGES_REQUESTED** [review]
Looks great. Just a small typo and one comment for you.

File: gallery/tutorials/generating_volume_projections.py (hunk 2 above)
Comment (garrettwrong) [reply gallery/tutorials/generating_volume_projections.py:114]: Yeah, that's a good Idea. I forgot all about that option 😅 .

Comment (garrettwrong) [reply gallery/tutorials/generating_volume_projections.py:114]: Turns out the figsize is clamped by the gallery, but I put a different tweak.  Default was 5 columns, but I changed to min(5, n_images)... so in this case 3 columns.  Image are slightly bigger, and centered, now.

Comment (j-c-c) [reply gallery/tutorials/generating_volume_projections.py:114]: Nice. Yeah, I was wondering about the choice of 5 columns. Looks good.

**Review by j-c-c — APPROVED** [review]

**Comment by garrettwrong** [comment]
Cool, thanks.  Since this primarily just adds a tutorial doc I think we can relieve Joakim from reviewing this one too.  We can always give it another go.

**Pull request merged**